src/components/Alert/alert.tsx

```diff
@@ -0,0 +1,94 @@
+import { useEffect, useState } from "react";
+
+type ErrorAlertProps = {
+    message: string;
+    dismiss: "manual" | "auto";
+    dismissTime?: number;
+    type: "Error" | "Success" | "Info" | "Warning";
+
+}
+
+
+function Alert({ message, dismiss, dismissTime, type }: ErrorAlertProps) {
+    const [show, setShow] = useState(true);
+    const [fadeOut, setFadeOut] = useState(false);
+
+    useEffect(() => {
+        if (dismiss === "auto") {
+            if (dismissTime) {
+                const timer = setTimeout(handleClose, dismissTime);
```

Comment:
this does the same thing, consider
```tsx
const time = dismissTime || 5000;
```

In the case where you want to support zero - 
```tsx
const time = dismissTime === 0 ? 0 : dismissTime || 5000;
```